src/incubator/expandableOverlay/index.tsx

```diff
@@ -92,7 +97,12 @@ const ExpandableOverlay = (props: ExpandableOverlayProps, ref: any) => {
 
   const renderOverlay = () => {
     if (renderCustomOverlay) {
-      return renderCustomOverlay({expandableVisible, openExpandable: showExpandable, closeExpandable: hideExpandable});
+      return renderCustomOverlay({
+        visible: expandableVisible,
```

Comment:
Why not name it as `visible` internally as well?

Reply:
Changed